db/migrate/20170817064448_create_tenant_api_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTenantApiRequests < ActiveRecord::Migration
+  def change
+    create_table :tenant_api_requests do |t|
+      t.integer :tenant_id, null: false
+      t.integer :count, default: 0
```

Comment:
We have a many to many relationship from `Tenant <-> TenantApiRequest`, but each `TenantApiRequest` has a `count`. Can you explain the purpose of this `count` attribute?